distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -322,6 +322,9 @@ Provides a list of environment variables.
 |`spatial`
 |Searches spatially the local Catalog.
 
+|`validate`
+|Runs the specified file against all installed validators.
```

Comment:
"Runs" might be too generic a verb here. Does it only work on XML files? If so, maybe something like "Validates an XML file against all installed validators." Or "Prints validation errors and warnings generated by an XML file". Does it work on whole directories? If so, maybe "Prints validation errors and warnings generated by an XML file or a directory of XML files".